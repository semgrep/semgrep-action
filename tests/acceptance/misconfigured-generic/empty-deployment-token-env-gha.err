=== Running: semgrep ci --no-suppress-errors
