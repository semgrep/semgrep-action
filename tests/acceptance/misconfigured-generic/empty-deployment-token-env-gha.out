run `semgrep login` before using `semgrep ci` or set `--config`
